src/components/Table/TableBody/RowActionsCell/RowActionsCell.jsx

```diff
@@ -126,8 +121,25 @@ const onClick = (e, id, action, onApplyRowAction) => {
 class RowActionsCell extends React.Component {
   state = {
     isOpen: false,
+    ltr: true,
   };
 
+  componentDidMount() {
+    // this is latest changes
+    if (document.dir === 'rtl') {
+      this.setState(state => ({ ltr: !state.ltr }));
+    }
+  }
+
+  componentDidUpdate(prevProp, prevState) {
+    // this is latest changes
+    const { ltr } = this.state;
+    const isLtr = document.dir === 'ltr';
+    if (prevState.ltr !== isLtr) {
+      this.setState(state => ({ ltr: !state.ltr }));
+    }
```

Comment:
@davidicus CI is failing here

```
/home/travis/build/IBM/carbon-addons-iot-react/src/components/Table/TableBody/RowActionsCell/RowActionsCell.jsx
  136:13  error  'ltr' is assigned a value but never used  no-unused-vars
```

should it be this?

```suggestion
    const { ltr } = this.state;
    const isLtr = document.dir === 'ltr';
    if (prevState.ltr !== isLtr) {
      this.setState(state => ({ ltr: !ltr }));
    }
```

or were you intending to use prevState.ltr instead?

Reply:
I think the linter is complaining about the unused variable. Removing and will see if travis passes.